### DISCUSSION TITLE
xdm UI almost black after 2x scale on Linux

### DISCUSSION THREAD

**Discussion by YourFavotireAgentRUN** [subhra74/xdm#757] [General]
My screen resolution is 2560x1440. After installing XDM, the UI was too small and hard to click. Now, I can't see anything.
[Screenshot](https://postimg.cc/9rKsGd7M)

Laptop asus G14
Elementary OS and Zorin 16 (same problem)
Garuda Linux (UI too small)

**Comment by hoishing** [comment]
same here on Zorin 16, need to switch the zoom back to 100% in order to use it. Edit and set `zoomLevelIndex:3` in `~/.xdman/config.txt`, where 3 means 100%

**Reply by jason-eu** [reply]
I kept re-installing and looking around for a config file and came here, you're a life saver mate!
